sound/soc/intel/boards/sof_sdw.c

```diff
@@ -187,6 +187,15 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_RT715_DAI_ID_FIX |
 					SOF_SDW_FOUR_SPK),
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Intel_adlrvp"),
```

Comment:
can we use the same match criteria as other platforms

````
			DMI_MATCH(DMI_SYS_VENDOR, "Intel Corporation"),
			DMI_MATCH(DMI_PRODUCT_NAME, "Ice Lake Client"),
````

Reply:
The dmidecode on adlrvp didnt show vendor as INTEL. hence used the family. will talk to BIOS team and see if that can be fixed soon.

Reply:
this will be done by coreboot team. so i can add this information and post v2 by EOD

Reply:
@plbossart with coreboot being used, the DMI match for vendor wont be possible as suggested. The corresponding coreboot.org conversation for reference :
https://review.coreboot.org/c/coreboot/+/51408/comment/3a850ca7_f58f7f79

Hence i think we should go ahead with current FAMILY string or i can change to product name ONLY. 

Let me know your thoughts and i can publish v2 as per that.

Reply:
sorry, I don't get the point at all. There is no reason why an Intel board should be named differently because of coreboot/UEFI changes.

@kv2019i what are the other RVPs using with the UEFI BIOS?

Reply:
@plbossart found the fix. now we can use Intel and adlrvp as match. posting v2 now

Reply:
For ADL UEFI BIOS,
```
			DMI_MATCH(DMI_SYS_VENDOR, "Intel Corporation"),
			DMI_MATCH(DMI_PRODUCT_NAME, "Alder Lake Client Platform"),
```

Reply:
@sathyap-chrome Can you please revisit why we need to have difference DMI names for CoreBoot? that's not good. Fix Coreboot!

Reply:
@fredoh9 can you try my patch and confirm it works with UEFI. let me talk to coreboot engineer on why there is a difference.